src/participant_network.star

```diff
@@ -34,6 +34,9 @@ CL_NODE_STARTUP_TIME = 5 * time.second
 
 CL_CLIENT_CONTEXT_BOOTNODE = None
 
+def zfill_custom(value, width):
```

Comment:
can this function be at the end of this file? the public cares about the funciton below it while this is just a util

Reply:
fixed